src/main/java/hudson/plugins/sidebar_link/LinkAction.java

```diff
@@ -74,7 +74,7 @@ public String getUrlName() {
     public String getIconFileName() { return icon; }
     
     @Restricted(NoExternalUse.class)
-    public String getUnprotectedLink() {
+    public String getUnprotectedUrlName() {
```

Comment:
I accidentally rebased this branch on master after that PR was merged, this is the diff from #8